### PR TITLE
pages: added horizon static page

### PR DESCRIPTION
**Pull request by 0einstein0**
Closes Issue [#706](https://github.com/zenodo/zenodo-rdm/issues/706)

Screenshot 2024-02-15 at 01 25 48

### CODE REVIEW

File: app_data/pages.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+- url: /communities/horizen/info # The pid_value i.e. (horizen) to be replaced according to the respective branded community
+  title: HorizonZen - Information
```
Comment (alejandromumo) [inline app_data/pages.yaml:2]: question: does it need quotes?

File: app_data/pages.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+- url: /communities/horizen/info # The pid_value i.e. (horizen) to be replaced according to the respective branded community
```
Comment (alejandromumo) [inline app_data/pages.yaml:1]: question: where does the `horizen` value came from?

File: templates/themes/horizon/invenio_app_rdm/default_static_page.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
```
Comment (alejandromumo) [inline templates/themes/horizon/invenio_app_rdm/default_static_page.html:3]:
```suggestion
# Copyright (C) 2024 CERN.
```

File: templates/themes/horizon/invenio_app_rdm/default_static_page.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+{% extends "invenio_communities/base.html" %}
+
+{% set title = page.title %}
+{% set community_menu_active = True %}
+{% block page_body %}
+	{% include "invenio_communities/details/header.html" %}
+<div class="ui container rel-mt-2">
```
Comment (alejandromumo) [inline templates/themes/horizon/invenio_app_rdm/default_static_page.html:25]: nit: can you format the code? easier to read.

**Comment by alejandromumo** [comment]
Reviewed with @yashlamba

File: app_data/pages.yaml (hunk 2 above)
Comment (0einstein0) [reply app_data/pages.yaml:1]: it is the pid_value of the themed community on my local

File: app_data/pages.yaml (hunk 1 above)
Comment (0einstein0) [reply app_data/pages.yaml:2]: no, even the template doesn't, I'll remove it.

**Review by anikachurilova — APPROVED** [review]

**Pull request merged**